frontend/src/pages/upload/UploadPage.tsx

```diff
@@ -25,6 +25,7 @@ export const UploadPage: FunctionComponent = (): ReactElement => {
   const methodologiesLoadable = useRecoilValueLoadable(methodologiesAtom);
   const [excelFile, setExcelFile] = useRecoilState(excelFileAtom);
 
+  // TODO reset form field on click Add another
```

Comment:
Yeah, I even noticed it during testing, but decided that it's not too critical :)